app_module/src/main/java/com/hmos/compat/app/AlertDialog.java

```diff
@@ -0,0 +1,254 @@
+/*
```

Comment:
Incorrect copyright!! Please add the applib group copyright for the files developed from scratch.